src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -925,3 +925,78 @@ def build_vm_daExtensionName_resource(_, vm_name, location):
     daExtensionName_resource['location'] = location
     daExtensionName_resource['dependsOn'] = ['Microsoft.Compute/virtualMachines/{0}/extensions/OMSExtension'.format(vm_name)]  # pylint: disable=line-too-long
     return daExtensionName_resource
+
+
+# used for creating default monitor alert rules
+def build_vm_alert_rules_resource(cmd, resource_group_name, vm_name):
+    from azure.cli.core.commands.client_factory import get_subscription_id
+    from msrestazure.tools import resource_id
+    import copy
+    vm_alert_rule_resources = []
+    vm_id = resource_id(
+        subscription=get_subscription_id(cmd.cli_ctx),
+        resource_group=resource_group_name,
+        namespace='Microsoft.Compute',
+        type='virtualMachines',
+        name=vm_name
+    )
+    alert_rule_resource_template = {
```

Comment:
Where do these constant values come from? Can it be specified by users? Maybe we can extract constant values as variables so that it can be configured more easily.

Reply:
This PR is to enable default alert for users. We're still discussing about the specific value with monitor team. User can use `az monitor metric alert` to config these values.